examples/benchmarking/plot_csv_file.py

```diff
@@ -3,9 +3,9 @@
 from dataclasses import dataclass, field
 from typing import Optional
 
+import matplotlib.pyplot as plt
```

Comment:
I had problems with isort on this file as well :D I think you might just reverse this change to fix isort

Reply:
Done ! And thanks for the hint :smile: